src/core_inner/backend/pearl/group.rs

```diff
@@ -308,8 +308,9 @@ impl PearlGroup {
                         warn!("pearl closed: {:?}", e);
                     }
                 }
+                let pearl = pearl.clone();
```

Comment:
Is it possible to find pearl and remove it, to prevent clone?

Reply:
I'll check

Reply:
Done